### PR TITLE
[System Tests]: Adding the 12 tests that were in my stash

### PR DESCRIPTION
**Pull request by wavesoft**
This PR adds all the tests that I had in my stash (some of them might be duplicates) I just wanted to commit everything just in case you want to do any bookkeeping while I am in the plane. ✈️ 

_Don't get intimidated by the many code lines, the tests are quite easy to read through._

- Jobs: Create a simple job (DCOS-12806)
- Jobs: Create a job with docker config (DCOS-12828)
- Jobs: Create a job with labels (DCOS-12827)
- Jobs: Create a job with schedule (DCOS-12826)
- Apps: Create a simple app (DCOS-12807)
- Apps: Creates an app with artifacts (DCOS-12825)
- Apps: Creates an app with command health check (DCOS-12824)
- Apps: Create an app with docker config (DCOS-12823)
- Apps: Create an app with environment variables (DCOS-12812)
- Apps: Create an app with external volume (DCOS-12821)
- Apps: Create an app with HTTP health check (DCOS-12820)
- Apps: Create an app with labels (DCOS-12819)
- Apps: Create an app with persistent volume (DCOS-12818)
- Pods: Create a pod with multiple containers (DCOS-12809)

To test this, use:
<pre>
docker run -it --rm --ipc=host \
  -e CLUSTER_URL=<strong>$CLUSTER_URL</strong>  \
  -v `pwd`:/dcos-ui \
  mesosphere/dcos-ui dcos-system-test-driver \
  ./system-tests/driver-config/cluster.sh
</pre>

**Checklist**
- [x] Did you add a JIRA issue in a commit message or as part of the branch name?
- [ ] Did you add new unit tests?
- [ ] Did you add new integration tests?
- [ ] If this is a regression, did you write a test to catch this in the future?

### CODE REVIEW

**Comment by Poltergeist** [comment]
flicts 😄

**Comment by wavesoft** [comment]
Rebased 😎

**Comment by wavesoft** [comment]
The System Tests are passing, trust me
The tests are passing

**Comment by Poltergeist** [comment]
There is a flaky test:

`Timed out retrying: Expected to find content: 'Running' within the element: <tr> but never did.`

Basically what is happening is that the app got into delayed instead of running state. Maybe we should not check for the state but rather check if the right configuration has been deployed.

**Comment by wavesoft** [comment]
These are not the bugs you are looking for. Merge along, merge along...

The tests are passing

**Comment by nLight** [comment]
LGTM! 💯  💖

File: system-tests/services/test-pods.js [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+        .click();
+
+      // Verify the review screen
+      cy
+        .root()
+        .configurationSection('General')
+        .configurationMapValue('Service ID')
+        .contains(`/${Cypress.env('TEST_UUID')}/${serviceName}`);
+      //
+      // TODO: Uncomment this when this gets merged
```
Comment (nLight) [inline system-tests/services/test-pods.js]: Please don't forget ☝️

Comment (Poltergeist) [inline system-tests/services/test-pods.js]: Shouldn't we test this aswell?

**Comment by Poltergeist** [comment]
image

👍

**Comment by Poltergeist** [comment]
Tests seem to pass here is a label.

File: system-tests/services/test-pods.js (hunk 1 above)
Comment (wavesoft) [inline system-tests/services/test-pods.js]: Created DCOS-14972 to keep track of this 👍

**Comment by orlandohohmeier** [comment]
🚢

**Pull request merged**